src/s2wasm.h

```diff
@@ -1167,6 +1173,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (startFunction.size()) {
```

Comment:
Actually it's `!!startFunction` which is horrible but `.is()` doesn't do the right thing.